refarch-frontend/tsconfig.node.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "@tsconfig/node20/tsconfig.json",
+  "extends": "@tsconfig/node-lts/tsconfig.json",
```

Comment:
sonst willst du immer den exakten Hashwert angeben, warum hier nicht?

Reply:
Weil hier "der exakte Hashwert" eindeutig durch die Versionsnummer des npm-Pakets festgelegt wird. Die Hashwerte gibt es ja nur bei den Pipeline-Actions. Außerdem kann so die Pflege der `@tsconfig` Dependency vereinfacht werden, weil man nur die Versionsnummer bumpen und nicht die Versionsnummer austauschen muss. Man kann also weiterhin in den Genuss von Renovatebot kommen